roles/DCOS.master/tasks/main.yml

```diff
@@ -17,6 +17,13 @@
   notify: reload facts
 - meta: flush_handlers
 
+- name: Write CA service PSK
+  copy:
+    dest: "/root/.dcoscertstrap_psk"
+    mode: 0600
+    content: "{{ hostvars['psk_dummy_host']['dcoscertstrap_psk'] }}"
```

Comment:
as the variable change this will also change on every run

Reply:
This is intentional